src/components/Divider/index.js

```diff
@@ -0,0 +1,3 @@
+import Divider from './Divider';
```

Comment:
Now I understand :) But not really what I thought. I was rather thinking to add border to TotalPrice.

Reply:
Divider is used in multiple places in the app. TotalPrice is one of 10.